src/enums/text-format.enum.ts

```diff
@@ -0,0 +1,8 @@
+export enum TextFormats {
+  BOLD = 'bold',
+  ITALIC = 'italic',
+  ITALIC_BOLD = 'italic bold',
```

Comment:
Values with spaces is usually very bad.
Unless you are sure of you I suggest you to turn it to kebab-case.

Reply:
Ok, I wasn't sure... :stuck_out_tongue: